docs/pages/_data/authors.yml

```diff
@@ -0,0 +1,6 @@
+# https://github.com/Phlow/feeling-responsive/blob/gh-pages/_data/authors.yml
+# http://blog.sorryapp.com/blogging-with-jekyll/2014/02/06/adding-authors-to-your-jekyll-site.html
+ccpi-devs:
+  name: CCPi Developers
```

Comment:
CCPi or CIL?

Reply:
I just copied the `setup.py.in` file. Changed everywhere now.